teamwork/apps/projects/views/EditProjectView.py

```diff
@@ -233,17 +110,169 @@ def edit_project(request, slug):
             # Not sure if view_one_project redirect will work...
             return redirect(view_one_project, project.slug)
     else:
-        form = EditProjectForm(request.user.id, instance=project, members=members)
-
-        # TEMPORARILIY COMMENTED OUT, DUE TO JULLIG HAVING TROUBLE ADDING MEMBERS
-        # if members:
-        #     form.fields['project_owner'].required = True
-        #     form.fields['scrum_master'].required = True
+        form = EditProjectForm(request.user.id, instance=project, members=members)        
 
     return render(request, 'projects/edit_project.html', {'page_name': page_name,
         'page_description': page_description, 'title' : title, 'members':members,
         'form': form, 'project': project, 'user':request.user})
 
+def add_member_to_project(request, project):
```

Comment:
If i'm understanding this right, instead of redirecting to a project page, we just use an alert?